From 65ccfd9274d1fdd36801c4a903c1de35fcded080 Mon Sep 17 00:00:00 2001
From: Tong <hetongapp@gmail.com>
Date: Sat, 20 Sep 2025 18:39:20 -0400
Subject: [PATCH] revise user to codesys

---
 codemeter | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/codemeter b/codemeter
index 8f634fe..de30b6f 100644
--- a/codemeter
+++ b/codemeter
@@ -19,7 +19,7 @@ DESC="CodeMeter Server"
 NAME=codemeter
 DAEMON=/usr/sbin/CodeMeterLin
 DAEMON_ARGS="-f -v"
-USER=root
+USER=codesys
 SCRIPTNAME=/home/codesys/$NAME
 CONFIGFILE=/etc/wibu/CodeMeter/Server.ini
 
-- 
2.50.1

